apps/src/music/views/MusicView.jsx

```diff
@@ -271,6 +285,10 @@ class UnconnectedMusicView extends React.Component {
 
   stopSong = () => {
     this.player.stopSong();
+
+    this.isExecutingForPlay = false;
```

Comment:
Is this call needed? `executeCompiledSong` will assign this as well right?

Reply:
Good catch!